src/docs/assets/highlight.css

```diff
@@ -0,0 +1,22 @@
+:root {
```

Comment:
I will add the instructions in README. Use the command npx typedoc --out src/docs src/src to generate. I looked into auto generating docs, doesn't seem to be an easy way to do this.

Reply:
Also seems to be standard to keep docs near source code? Wasn't sure how to do this given we have nested src folders.

Reply:
Yes, keeping docs near source code is a good practice. But, since the docs are literally generated from the source code, the typings in the source code and the comments in the source code, it's fair to say that the source code itself is documentation.

typedoc essentially just extract all the comments and typing and puts that into neat HTML pages. This may be useful for people new to the codebase or those who want to get an overview of the code without getting dirty (i.e, a GTA)